src/commands/admin/jobs/jobs.js

```diff
@@ -16,7 +16,7 @@ module.exports = ({
         const status = (entry ? entry.jobState : job.meta.enabled) ? ':white_check_mark:' : ':x:'
 
         return {
-          "name": `${status} - \`${job.meta.name}\` | ${secsToDHMS(job.meta.interval / 1000)}`,
+          "name": `${status} - \`${job.meta.name}\` | ${secsToDHMS(job.meta.defaultInterval ? job.meta.defaultInterval / 1000 : job.meta.interval / 1000)}`,
```

Comment:
The logic here seems incorrect. We should be defaulting to `job.meta.defaultInterval` if the row in the database (`jobSchema`, on the `entry` variable) does not exist. I'd expect the following (not tested):

```js
const entry = jobStatus.find((s) => s.jobName == job.meta.name)
const status = (entry ? entry.jobState : job.meta.enabled) ? ':white_check_mark:' : ':x:'
// below checks to see if 'entry' exists, if so, check if jobInterval is null, if not use the interval from db row, otherwise use default
const interval = (entry ? (entry.jobInterval ? entry.jobInterval : job.meta.defaultInterval) : job.meta.defaultInterval

return {
  "name": `${status} - \`${job.meta.name}\` | ${secsToDHMS(interval / 1000)}`,
  ...
}
```

Reply:
Yes, however I didn't set `defaultInterval` for all of the jobs, since the ones not controlled by the guild would have an ` interval` and `defaultInterval` set to same values. Here's needed an additional check, whether ` interval` is `undefined`, if so, the value from `job.meta.interval` should be used.